pkg/controller/core.oam.dev/v1alpha2/application/revision.go

```diff
@@ -48,40 +47,25 @@ import (
 )
 
 const (
-	// ConfigMapKeyResources is the key in ConfigMap Data field for containing data of resources
-	ConfigMapKeyResources = "resources"
+	// ConfigMapKeyComponents is the key in ConfigMap Data field for containing data of components
+	ConfigMapKeyComponents = "components"
+	// ConfigMapKeyPolicy is the key in ConfigMap Data field for containing data of policies
+	ConfigMapKeyPolicy = "policies"
 )
 
 func (h *AppHandler) createResourcesConfigMap(ctx context.Context,
```

Comment:
will this refactor afftect the `AppDeployment` logic? @hongchaodeng

Reply:
No.